src/main/java/org/jabref/logic/database/DuplicateCheck.java

```diff
@@ -65,6 +64,7 @@ public class DuplicateCheck {
         DuplicateCheck.FIELD_WEIGHTS.put(StandardField.NOTE, 0.1);
         DuplicateCheck.FIELD_WEIGHTS.put(StandardField.COMMENT, 0.1);
         DuplicateCheck.FIELD_WEIGHTS.put(StandardField.DOI, 3.);
+        DuplicateCheck.FIELD_WEIGHTS.put(StandardField.ISBN, 2.5);
```

Comment:
Does the modification also work without this addition? Reading the comment, this adds some duplicate check for articles having the same ISBN.

I think, it's good to keep, but I am wondering... :)

Reply:
Yes it works without this addition. Should I remove it for now?

Reply:
Yes, remove it. We can add if we have more tests for similar entries... - I think, if entries are different, the same ISBN could make it the same. However, I am not sure about the algoirthm. Is it typos at the author? If same ISBN, the entries are same? If no ISBN, not same? That feels strange. -- Thus, remove.